Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }
 }

```

Comment:
**Suggestion**: Ensure proper cleanup of resources, such as closing the Scanner instance, to prevent resource leaks. Consider implementing a try-with-resources statement if applicable. <br> **Code Suggestion**: 
 ```
 +        sc.close();
 ```